src/js/HRModules/showForms.js

```diff
@@ -0,0 +1,183 @@
+            child.innerHTML = form.questions.length;
+            div.appendChild(child);
+
+            child = document.createElement('div');
+
+            let img = new Image();
+            img.src = eyePng;
+            img.onclick = () => {
+                this.show(form);
+            };
+            child.appendChild(img);
+
+            div.appendChild(child);
+
+            child = document.createElement('div');
+
+            img = new Image();
+            img.src = pencilPng;
+            child.appendChild(img);
+
+            div.appendChild(child);
+
+            child = document.createElement('div');
+
+            img = new Image();
+            img.src = deletePng;
+            child.appendChild(img);
+
+            div.appendChild(child);
+
+            $id('showForms-list-table').appendChild(div);
+        }
+
+        this.showAll();
+
+        $id('showForms-content-loading').remove();
+    },
+
+    open() {
+        if (this.init)
+            this.init();
+        else {
+            //
+        }
+    },
+
+    showAll() {
+        $id('showForms-list').style.display = 'block';
+        $id('showForms-form').style.display = 'none';
+    },
+
+    show(which) {
+        $id('showForms-list').style.display = 'none';
+        $id('showForms-form').style.display = 'block';
+
+        $id('showForms-form-title').innerHTML = which.title;
+        $id('showForms-form-content').innerHTML = '';
+
+        for (const question of which.questions) {
+            if (question.type.toLowerCase() === 'o') {
+                $id('showForms-form-content')
+                    .appendChild(this.createOpenQuestion(question.number, question.content));
+            } else if (question.type.toLowerCase() === 'w') {
+                $id('showForms-form-content')
+                    .appendChild(
+                        this.createClosedQuestion(question.number,
+                            question.content,
+                            question.answers)
+                    );
+            } else if (question.type.toLowerCase() === 'l') {
+                $id('showForms-form-content')
+                    .appendChild(this.createNumberQuestion(question.number, question.content));
+            }
+        }
+    },
+
+    createOpenQuestion(number, question) {
+        const div = document.createElement('div');
+        div.classList.add('showForms-form-question');
+        div.classList.add('showForms-form-oQuestion');
+
+        const p = document.createElement('p');
+        p.innerHTML = `${number}. ${question}`;
+        div.appendChild(p);
+
+        const input = document.createElement('input');
+        input.placeholder = 'Your answer';
+        div.appendChild(input);
+
+        return div;
+    },
+
+    createClosedQuestion(number, question, answers) {
+        const div = document.createElement('div');
+        div.classList.add('showForms-form-question');
+        div.classList.add('showForms-form-wQuestion');
```

Comment:
Jak powyżej

Reply:
To co ma być w reszcie?
o - open
w - ?
l - ?